raycicmd/config.go

```diff
@@ -51,6 +55,9 @@ type config struct {
 
 	// SkipTags is the list of tags that will always be skipped.
 	SkipTags []string `yaml:"skip_tags"`
+
+	// Additional docker plugin configs.
+	DockerPlugin *dockerPluginConfig `yaml:"docker_plugin"`
```

Comment:
It's a bit unclear which field in this list is required vs optional; maybe put (Optional) for this field in the comment.

Also I get the impression that ForgeDirs is not used when I create the config for release test pipeline.

Reply:
yeah. most fields are optional, only some fields are mandatory.

forge dir is deprecated now after we added the wanda rules. Let me file a PR to remove that.

Reply:
added many comments. will remove the forgedir in a follow up.